tests/test_using.py

```diff
@@ -171,6 +212,46 @@ def test_create_batch_custom_base(self):
             self.assertEqual(obj.id, 2)
             self.assertEqual(obj.foo, 'bar')
 
+    def test_create_async(self):
+        loop = asyncio.get_event_loop()
+        obj = loop.run_until_complete(factory.create_async(FakeAsyncModel, foo='bar'))
+        self.assertEqual(obj.id, None)
```

Comment:
`id` is `None` because `create()` was not called.

Is that the expected behavior? I would expect that when the task to create the object is done, the object has been recorded to the database and has an id?

Reply:
It's expected because the `make_factory` function will output a factory based on Factory class.
So we end up with model_class(**kwargs) to create the object,  not the model async `create` function.
I'll add comments to the test.

For comparison, the sync test:
```
    def test_create(self):
        obj = factory.create(FakeModel, foo='bar')
        self.assertEqual(obj.id, None)
        self.assertEqual(obj.foo, 'bar')
```

If you look at the test `test_create_async_custom_base`, it has parameter `FACTORY_CLASS=FakeAsyncModelFactory` and the async creation is ok.